.github/workflows/node-zxcron-citr-extended-test-suite.yaml

```
@@ -0,0 +1,153 @@
+        uses: actions/setup-java@387ac29b308b003ca37ba93a6cab5eb57c8f5f93 # v4.0.0
+        with:
+          distribution: "temurin"
+          java-version: "21"
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@dbbdc275be76ac10734476cc723d82dfe7ec6eda # v3.4.2
+        with:
+          cache-read-only: false
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 20
+
+      - name: Compile
+        id: gradle-build
+        run: ${GRADLE_EXEC} assemble :test-clients:yahCliJar --scan
+
+      - name: Spotless Check
+        if: ${{ !cancelled() }}
+        run: ${GRADLE_EXEC} spotlessCheck --scan
+
+      - name: Timing Sensitive Tests
+        id: gradle-timing-sensitive
+        if: ${{ steps.gradle-build.conclusion == 'success' && !cancelled() }}
+        run: ${GRADLE_EXEC} timingSensitive --continue --scan
+
+      - name: Publish Unit Test (Timing Sensitive) Report
+        uses: step-security/publish-unit-test-result-action@4519d7c9f71dd765f8bbb98626268780f23bab28 # v2.17.0
+        if: ${{ inputs.enable-unit-tests && steps.gradle-build.conclusion == 'success' && !cancelled() }}
+        with:
+          check_name: 'Node: Timing Sensitive Unit Test Results'
+          json_thousands_separator: ','
+          junit_files: "**/build/test-results/timingSensitive/TEST-*.xml"
+          comment_mode: errors # only comment if we could not find or parse the JUnit XML files
+
+      #      - name: Time Consuming Tests
+      #        id: gradle-time-consuming
+      #        if: ${{ steps.gradle-build.conclusion == 'success' && !cancelled() }}
+      #        run: ${GRADLE_EXEC} timeConsuming --continue --scan
+      #
+      #      - name: Publish Unit Test (Time Consuming) Report
+      #        uses: step-security/publish-unit-test-result-action@4519d7c9f71dd765f8bbb98626268780f23bab28 # v2.17.0
+      #        if: ${{ steps.gradle-build.conclusion == 'success' && !cancelled() }}
+      #        with:
+      #          check_name: 'Node: Time Consuming Unit Test Results'
+      #          json_thousands_separator: ','
+      #          junit_files: "**/build/test-results/timeConsuming/TEST-*.xml"
+      #          comment_mode: errors # only comment if we could not find or parse the JUnit XML files
+
+      - name: Hammer Tests
+        id: gradle-hammer-tests
+        if: ${{ steps.gradle-build.conclusion == 'success' && !cancelled() }}
+        run: ${GRADLE_EXEC} hammer --continue --scan
+
+      - name: Publish Hammer Test Report
+        uses: step-security/publish-unit-test-result-action@4519d7c9f71dd765f8bbb98626268780f23bab28 # v2.17.0
+        if: ${{ steps.gradle-build.conclusion == 'success' && !cancelled() }}
+        with:
+          check_name: 'Node: Hammer Test Results'
+          json_thousands_separator: ','
+          junit_files: "**/test-clients/build/test-results/hammer/TEST-*.xml"
```

```suggestion
          junit_files: "**/build/test-results/hammer/TEST-*.xml"
```
`hammer` may exist in any project